tests/test_data.py

```diff
@@ -188,6 +200,22 @@ def test_indexing_into_sparse(self):
         sf.data[1:-1, 0] = np.arange(8)
         assert np.all(sf.data[1:-1, 0] == np.arange(8))
 
+    @pytest.mark.parametrize('idx, expected', [
+        ('(slice(10, None, -1), slice(11, None, -3))',
+         '(slice(0, 11, 1), slice(2, 12, 3))'),
+        ('(2, 5)', '(slice(2, 3, 1), slice(5, 6, 1))')
+    ])
+    def test_local_data_idx(self, idx, expected):
```

Comment:
This probably needs its own class, as it's a test for a support function, not for an actual Data